jwala-services/src/main/java/com/cerner/jwala/service/jvm/impl/JvmServiceImpl.java

```diff
@@ -622,7 +638,15 @@ private void secureCopyJvmConfigJar(Jvm jvm, String jvmConfigJar, User user) thr
 
     private void deployJvmConfigJar(Jvm jvm, User user, String jvmJar) throws CommandFailureException {

```

Comment:
My bad I missed some of the code reviews. I don't think this should be in a state service, please ask @akinny if we can transfer this